pyFAST/input_output/tests/example_files/FASTIn_TurbSim.inp

```diff
@@ -0,0 +1,74 @@
+---------TurbSim v2.00.* Input File------------------------

```

Comment:
I'd suggest to rename this file with the extension `.dat`

Reply:
The extension of TurbSim's input file is .inp. why should change?
![20211216144300](https://user-images.githubusercontent.com/42032942/146321351-19368122-202a-435f-9791-f41c4c5f682c.png)

Reply:
I think this extension is a bit old, and unnecessary, that's why I'm trying to push for people to use `.dat.

Reply:
okay, get it！I think your suggestion is very helpful！